publish/publish_workflow.wdl

```diff
@@ -0,0 +1,667 @@
+        File githubTokenFile
+        String githubUser
+        String githubName
+        String githubEmail
+        String organization
+        Int releaseVersion
+        String publishDocker
+    }
+
+    parameter_meta {
+        releaseVersion: "Current version being released"
+        organization: "Organization on which the release will be performed. Can be used for testing."
+    }
+
+    Int nextV = releaseVersion + 1
+    Boolean useEmailForName = githubName == "null"
+
+    command <<<
+        # Do not use `set -x` or it will print the GitHub token!
+        set -euo pipefail
+
+        echo 'Setup Git'
+        # https://git-scm.com/book/en/v2/Git-Tools-Credential-Storage
+        echo "https://~{githubUser}:$(cat ~{githubTokenFile})@github.com" > githubCredentials
+        git config --global credential.helper "store --file $PWD/githubCredentials"
+        git config --global user.email ~{githubEmail}
+        ~{if (useEmailForName) then "" else "git config --global user.name '~{githubName}'"}
+
+        echo 'Clone repo and checkout develop'
+        git clone https://github.com/~{organization}/cromwell.git --branch develop cromwell
+        cd cromwell
+
+        echo 'Merge develop into master'
+        git checkout --track origin/master
+        git merge --no-edit develop
+
+        echo 'Tag the release'
+        git tag --message=~{releaseVersion} ~{releaseVersion}
+
+        echo 'Assemble jar for cromwell'
+        sbt -Dproject.version=~{releaseVersion} -Dproject.isSnapshot=false server/assembly womtool/assembly
+
+        echo 'Create and push the hotfix branch'
+        git checkout -b ~{releaseVersion}_hotfix
+
+        echo 'Update develop to point to next release version'
+        git checkout develop
+        sed -i -e '/cromwellVersion[[:space:]]=/s/[0-9][0-9]*/~{nextV}/' project/Version.scala
+        git add project/Version.scala
+        git diff-index --quiet HEAD \
+        || git commit --message "Update cromwell version from ~{releaseVersion} to ~{nextV}"
+
+        echo 'Push branches and tags'
+        git push origin master develop ~{releaseVersion} ~{releaseVersion}_hotfix
+    >>>
+
+    output {
+        File cromwellJar = "cromwell/server/target/scala-2.12/cromwell-~{releaseVersion}.jar"
+        File womtoolJar = "cromwell/womtool/target/scala-2.12/womtool-~{releaseVersion}.jar"
+    }
+
+    runtime {
+        docker: publishDocker
+        cpu: 4
+        memory: "8GB"
+    }
+}
+
+# doMinorRelease simply builds jars from the hotfix branch, tags the branch and push the tag to github
+task doMinorRelease {
+    input {
+        File githubTokenFile
+        String githubUser
+        String githubName
+        String githubEmail
+        String organization
+        String releaseVersion
+        String publishDocker
+    }
+
+    Float releaseVersionAsFloat = releaseVersion
+    Int majorReleaseNumber = floor(releaseVersionAsFloat)
+    String hotfixBranchName = "~{majorReleaseNumber}_hotfix"
```

Comment:
not used

Reply:
Bad copypasta. Its usage shall return shortly.